tests/Makefile

```diff
@@ -26,7 +26,8 @@ guess_platform_script := \
 	esac
 guess_platform := $(shell $(guess_platform_script))
 
-bin_dir ?= $(top_srcdir)/$(guess_platform)-bin
+#bin_dir ?= $(top_srcdir)/$(guess_platform)-bin
+bin_dir ?= $(top_srcdir)/_build/mac/Debug
```

Comment:
You'll need to figure out what to do with this, or it'll break running the tests on non-Mac.